java/mtsj/core/src/main/resources/config/application.properties

```diff
@@ -32,9 +32,13 @@ mythaistar.hourslimitcancellation=1
 
 ##Mailing properties
 #enabled set to false will use the MailMock service
-spring.mail.enabled=false
-spring.mail.host=smtp.gmail.com
-spring.mail.username=mythaistarrestaurant@gmail.com
-spring.mail.password=mythaistarrestaurant2501
-spring.mail.properties.mail.smtp.auth=true
-spring.mail.properties.mail.smtp.starttls.enable=true
\ No newline at end of file
+spring.mail.enabled=true
+#spring.mail.host=smtp.gmail.com
+spring.mail.host=de-nagios01.corp.capgemini.com
+spring.mail.port=25
+#spring.mail.username=mythaistarrestaurant@gmail.com
+#spring.mail.password=mythaistarrestaurant2501
+#spring.mail.properties.mail.smtp.auth=true
```

Comment:
Clean-Code: Remove outcommented stuff. It is all still in the git history and otherwise just spams the code.